ido/src/lib.rs

```diff
@@ -72,20 +115,27 @@ pub struct IDOContract{
     /// Ex: Project 1: Tickets [{Id: L1, Account Id: account1.testnet }, {Id: S2, Account Id: account2.testnet }, ...]
     /// The user tickets were stored here during re-calculate
     pub tickets_by_project: LookupMap<ProjectId, TicketAndAccountLookupMap>,
-    
+
     /// The list of projects that that account has registered whitelist.
     pub projects_by_account: LookupMap<AccountId, ProjectIdUnorderedSet>,
 
     /// The information of tiers that helps to identify the number of tickets to allocation to a specific user when they joined to a project
     //pub tiers: UnorderedMap<StakingTier, TierInfo>,
 
     pub test_mode_enabled: bool,
+
+    pub config: Config,
 }
 
 #[near_bindgen]
-impl IDOContract{
+impl IDOContract {
+    #[init]
+    pub fn new_with_default_config(owner_id: AccountId, staking_contract_id: AccountId, funding_ft_token_ids: Option<Vec<AccountId>>, test_mode_enabled: Option<bool>) -> Self {
+        Self::new(owner_id, staking_contract_id, funding_ft_token_ids, test_mode_enabled, Config::default())
+    }
+
     #[init]
-    pub fn new(owner_id: AccountId, staking_contract_id: AccountId, funding_ft_token_ids: Option<Vec<AccountId>>, test_mode_enabled: Option<bool>) -> Self {
+    pub fn new(owner_id: AccountId, staking_contract_id: AccountId, funding_ft_token_ids: Option<Vec<AccountId>>, test_mode_enabled: Option<bool>, config: Config) -> Self {
```

Comment:
The _config_ parameter should be an Option type. It could be used as the default value if not specified.